### ISSUE TITLE
Query : Will this bundle handle numerous attachments

### ISSUE THREAD

**Issue by asmafarhat2006** [fusonic/php-messenger-mailer-bundle#2]
HI 
Im facing an issue with Symfony mailer where attaching numerous files in the same email gives me Out of memory exception . 
Note my files are already persisted and saved way before sending the email
Im thinking of using this bundle to resolve this, but Im not sure if it will , it seems it still builds up those attachments in the same message? The bundle only makes sure they are persisted before sending them or am I missing something 

Any advice will be appreciated

**Comment by arjanfrans** [comment]
Hey. It _might_ resolve it, it depends on what your current setup looks like. Could you share your configuration related to email?

This bundle doesn't store the attachments inside the message, it writes them one by one to the filesystem (or somewhere else if you implement it).

**Comment by asmafarhat2006** [comment]
Hi
My email configuration is simple..uses symfony mailer and the attaches the
attachments.with the email. And the files are persisted before it sends the
email
Will your bundle attach the files with the email?

**Comment by arjanfrans** [comment]
Do you attach the files like this: https://symfony.com/doc/current/mailer.html#file-attachments

You could easily try out this bundle, it doesn't require any configuration.

**Comment by asmafarhat2006** [comment]
Yes I attach them this way.
Are the attachments going to be sent along with the email when I use your
bundle?
I cant use it straight away because Im using Symfony 5.4 and I ll also wont
need FileHandler since my files are already persisted on the file system

**Comment by arjanfrans** [comment]
Yes, the attachments will be there.

But since you are using the file system anyway, then you probably don't need this bundle. 

The purpose of this bundle is to abstract away the persisting (so for example you could also use a cloud storage for them) and to avoid that the actual message queue contains potentially large files.

**Comment by asmafarhat2006** [comment]
But even with the persisted files .whether I use attachFromPath or DataPart
I still get the memory leak. Since they are part of the message .

What I understood from your bundle .it separates the attachment data from
the main email message in the extended Email class.
Is that not correct?

**Comment by arjanfrans** [comment]
`attachFromPath` also doesn't store the attachment data. I think the problem lies elsewhere, how large are the files you are trying to attach.

**Comment by asmafarhat2006** [comment]
Yep , it only occurs when fileSize >= 5MB and I attach atleast 5 files
